x-pack/plugins/apm/public/components/app/TraceOverview/TraceList.tsx

```diff
@@ -30,23 +29,22 @@ interface Props {
 
 const traceListColumns: ITableColumn[] = [
   {
-    field: 'sample',
+    field: 'name',
     name: 'Name',
     width: '40%',
     sortable: true,
-    render: (transaction: Transaction) => (
-      <TooltipOverlay content={transaction.transaction.name}>
-        <StyledTransactionLink transaction={transaction}>
-          {transaction.transaction.name}
+    render: (name, group: ITransactionGroup) => (
+      <TooltipOverlay content={name}>
+        <StyledTransactionLink transaction={group.sample}>
+          {name}
         </StyledTransactionLink>
       </TooltipOverlay>
     )
   },
   {
-    field: 'sample',
+    field: 'sample.context.service.name',
```

Comment:
Woop, didn't know EUI supported this!

Reply:
Haha yep, EUI doesn't care, it just hands back that field string to you.